.github/workflows/build.yml

```diff
@@ -9,6 +9,12 @@ env:
     CARGO_TERM_COLOR: always
 
 jobs:
+    lint-markdowns:
+        name: Lint markdown files
+        runs-on: ubuntu-latest
+        steps:
+            - name: Lint markdown files 
+            uses: actionshub/markdownlint@main
```

Comment:
Add a warning to ignore. We know that in the Changelog there will be multiple same headers which will cause it to throw an error
https://github.com/markdownlint/markdownlint/issues/175